src/observable/ConnectableObservable.ts

```diff
@@ -39,6 +39,15 @@ export class ConnectableObservable<T> extends Observable<T> {
   refCount(): Observable<T> {
     return new RefCountObservable(this);
   }
+
+  /**
+   * This method is opened for `ConnectableSubscription`.
+   * Not to call from others.
+   */
```

Comment:
Actually, I was suggested `closeSubscription` in opposite to @saneyuki did in first way. My bad, and sorry @saneyuki for making redo work.